CHANGELOG.md

```diff
@@ -23,10 +23,11 @@
 ## Improvements
 
 - [Generator] Generator will fail, when it generates `.ts` files that have compilation errors.
+- [OData] Add warning for deep update attempts in OData v2. 
```

Comment:
The changes in this file need to be handled carefully (by resolving the merge conflict), since we released `1.28.2` last week.